### PR TITLE
docs: improve custom theme by umi

### PR DESCRIPTION
**Pull request by hengkx**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by hengkx** [comment]
lint 修复了吗

**Review by Kacrosby88 — APPROVED** [review]
I'm just learning

File: docs/react/customize-theme.en-US.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Note:
 
 ### Customize in Umi
 
-You can easily use [theme](https://umijs.org/config/#theme) field in [config/config.js](https://github.com/ant-design/ant-design-pro/blob/56e648ec14bdb9f6724169fd64830447e224ccb1/config/config.js#L45) (Umi) file of your project root directory if you are using [Umi](http://umijs.org/), which could be an object or a javascript file path.
+You can easily use [theme](https://umijs.org/config/#theme) field in `.umirc.ts` file of your project root directory if you are using [Umi](http://umijs.org/), which could be an object or a javascript file path.
```
Comment (afc163) [inline docs/react/customize-theme.en-US.md:78]: 推荐用 config/config

Comment (hengkx) [reply docs/react/customize-theme.en-US.md:78]: 现在 新建的 默认不就只有一个 `.umirc.ts` 文件吗？

Comment (afc163) [reply docs/react/customize-theme.en-US.md:78]: 可以写 

```
.umirc.ts 或 [config/config.ts](https://github.com/ant-design/ant-design-pro/blob/v5/config/config.ts)
```

Comment (hengkx) [reply docs/react/customize-theme.en-US.md:78]: > 推荐用 config/config
这个 umi官方有介绍吗？

Comment (afc163) [reply docs/react/customize-theme.en-US.md]: https://umijs.org/zh-CN/docs/config

**Pull request merged**